radix-engine/src/system/system_callback.rs

```diff
@@ -1381,28 +1554,13 @@ impl<C: SystemCallbackObject> KernelTransactionCallbackObject for System<C> {
             global_address_reservations.push(global_address_reservation);
         }
 
-        let intents = executable.intents();
-        let output = match intents {
-            ExecutableIntents::V1(intent) => {
-                let rtn = system_service.call_function(
-                    TRANSACTION_PROCESSOR_PACKAGE,
-                    TRANSACTION_PROCESSOR_BLUEPRINT,
-                    TRANSACTION_PROCESSOR_RUN_IDENT,
-                    scrypto_encode(&TransactionProcessorRunInputEfficientEncodable {
-                        manifest_encoded_instructions: intent.encoded_instructions.clone(),
-                        global_address_reservations,
-                        references: intent.references.clone(),
-                        blobs: intent.blobs.clone(),
-                    })
-                    .unwrap(),
-                )?;
-                let output: Vec<InstructionOutput> = scrypto_decode(&rtn).unwrap();
-                output
-            }
-            ExecutableIntents::V2(_) => {
-                unimplemented!();
-            }
-        };
+        let system_logic_version = system_service.system().versioned_system_logic;
+
+        let output = system_logic_version.execute_transaction(
```

Comment:
I don't disagree with this, but just want to check -- your thinking is that at Cuttlefish, we swap V1 transactions to use the V2 processor too?